buildConfig/tasks/browserify.js

```diff
@@ -5,25 +5,33 @@ import buffer from 'vinyl-buffer';
 import debowerify from 'debowerify';
 import babelify from 'babelify';
 import browserifyShim from 'browserify-shim';
+import es from 'event-stream';
 
 module.exports = function(gulp, $, reload, gutil) {
   return function() {
-    var b = browserify({
-      entries: paths.srcPaths.scripts + '/main.js',
-      debug: true,
-    });
+    var files = [
+      paths.srcPaths.scripts + '/main.js',
+      paths.srcPaths.scripts + '/satisfaction.js'
+    ];
+
+    var tasks = files.map(function (entry) {
+      var distFilename = entry.replace(paths.srcPaths.scripts + '/', '').replace('.js', '.min.js');
 
-    b.transform(debowerify);
-    b.transform(babelify);
-    b.transform(browserifyShim);
+      return browserify({ entries: entry })
+        .transform(debowerify)
+        .transform(babelify)
+        .transform(browserifyShim)
+        .bundle()
+        .pipe(source(distFilename))
+        .pipe(buffer())
+        .pipe($.sourcemaps.init({ loadMaps: true }))
+          .pipe($.uglify())
+          .on('error', gutil.log)
+        .pipe($.sourcemaps.write('./'))
+        .pipe(gulp.dest(paths.distPaths.scripts));
+    });
 
-    return b.bundle()
-      .pipe(source('scripts.min.js'))
-      .pipe(buffer())
-      .pipe($.sourcemaps.init({loadMaps: true}))
-        .pipe($.uglify())
-        .on('error', gutil.log)
-      .pipe($.sourcemaps.write('./'))
-      .pipe(gulp.dest(paths.distPaths.scripts));
+    return es.merge.apply(null, tasks);
```

Comment:
as far as I can tell, that will mush all your source files into one.  I want two.